### PR TITLE
Display all dates and times in local timezone. Used moment library for

### PR DESCRIPTION
**Pull request by raamb**
this
Jira issue SNET-1099
https://singularitynet.atlassian.net/browse/SNET-1099

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -90,7 +91,7 @@ const Home: NextPage = () => {
       );
 
       const airdropSchedules = airdropTimelines.flat().map((timeline) => ({
-        time: new Date(timeline.airdrop_window_timeline_date),
+        time: new Date(moment.utc(timeline.airdrop_window_timeline_date)),
```
Comment (ksridharbabuus) [inline pages/index.tsx:94]: We don't need separately new Date if we are using the moment library as it provides date functionality.

File: components/Registration/index.tsx [hunk 2]
```diff
@@ -110,14 +111,12 @@ const Registration: FunctionComponent<RegistrationProps> = ({
   const endDate = useMemo(
     () =>
       activeWindow?.airdrop_window_status === WindowStatus.REGISTRATION
-        ? new Date(
-            `${activeWindow?.airdrop_window_registration_end_period} UTC`
-          )
+        ? new Date(moment.utc(activeWindow?.airdrop_window_registration_end_period))
```
Comment (ksridharbabuus) [inline components/Registration/index.tsx:114]: Refer the comment below on Date object.

File: pages/index.tsx (hunk 1 above)
Comment (raamb) [reply pages/index.tsx:94]: moment.utc does not return a date object and hence did this

**Comment by raamb** [comment]
Not needed anymore

**Pull request closed without merging**